src/Components/Navbar/Navbar.css

```diff
@@ -2,7 +2,8 @@
   width: 100%;
   height: 5rem;
   border: 1px solid black;
-  position: relative;
+  position: fixed;
+  z-index: 9999;
```

Comment:
use proper values for z index, maybe make util classes/ variables for z index from 10 to 100 eg, z-10, z-20.
then use those. It will become easier to debug

Reply:
Actually, that z index is of no use. I forgot to remove that but thanks for the suggestion making util classes for z index would be better approach I will use that.